apps/native/app/src/screens/inbox/inbox.tsx

```diff
@@ -154,6 +157,10 @@ type Filters = {
   archived?: boolean
   bookmarked?: boolean
   subjectContains?: string
+  senderNationalId?: string[]
```

Comment:
should this be plural?

Reply:
The api parameter is in singular, that is the reason I kept it like that 🤷‍♀️ But I considered changing it 😄